src/util/bson/mongo_timestamp.hpp

```diff
@@ -24,17 +24,31 @@
 namespace realm {
 namespace bson {
 
-struct Datetime {
-    std::time_t seconds_since_epoch;
-    Datetime(time_t epoch);
+struct MongoTimestamp {
+
+    MongoTimestamp(const int64_t seconds, const int64_t increment);
```

Comment:
There isn't really a reason to put const on non-reference arguments like that. It has no effect when in the declaration. It has some effect on the definition (and they don't have to match in top-level const), but it is rarely useful, so rarely used.